### PR TITLE
Make looking up bank ids easier

### PR DESCRIPTION
**Pull request by lhstrh**
Each reactor can look up its bank id if it has one. This PR generalizes this capability by allowing reactors to find the bank id of any of its parents by specifying a `depth` parameter in the look up method.

### CODE REVIEW

File: src/core/reactor.ts [hunk 1]
```diff
@@ -149,11 +149,15 @@ export abstract class Reactor extends Component {
      * Return the location of the reactor instance in a bank,
```
Comment (hokeun) [inline src/core/reactor.ts:149]: Maybe we should update the comments as well?

File: src/core/reactor.ts [hunk 2]
```diff
@@ -1734,7 +1738,7 @@ export class App extends Reactor {
                 }
     
             } else {
-                console.log(">>>>>>>>>Postponed Scheduling of timer " + timer._getFullyQualifiedName())
+                //console.log(">>>>>>>>>Postponed Scheduling of timer " + timer._getFullyQualifiedName())
```
Comment (hokeun) [inline src/core/reactor.ts:1741]: Maybe it's a good time to delete this logging, but I'm fine with leaving it as is.

**Review by hokeun — APPROVED** [review]
Looks good to me! Just left minor comments.

File: src/core/reactor.ts (hunk 1 above)
Comment (lhstrh) [reply src/core/reactor.ts:149]: Yep. Will do!

**Pull request merged**